src/repository/user-repository.ts

```diff
@@ -0,0 +1,8 @@
+import db from './db';
+import {Repository, User, UserFilters} from "../types";
+
+export class UserRepository extends Repository {
+    async getUsers(filters: UserFilters): Promise<User[]> {
+        return db.query(`SELECT * FROM users WHERE location=${filters.location}`);
```

Comment:
Use parameters in place of string manipulation, you are introducing potential SQL injection security concern here.